### PR TITLE
Fix the path field in the backend log

### PR DESCRIPTION
**Pull request by alex-schneider**

### CODE REVIEW

File: logging/upstream_log.go [hunk 1]
```diff
@@ -61,15 +60,6 @@ func (u *UpstreamLog) RoundTrip(req *http.Request) (*http.Response, error) {
 		requestFields["bytes"] = req.ContentLength
 	}
 
-	path := &url.URL{
```
Comment (johakoch) [inline logging/upstream_log.go:64]: Looks like the `path` log field also contained a query. With the fix this looks no longer the case. I'm not sure what the intended behaviour is.

Comment (alex-schneider) [reply logging/upstream_log.go:64]: The `path` has not to contain the `query` string. The `query` is a part of the `url` field in the log.

File: server/testdata/integration/api/08_couper.hcl [hunk 2]
```diff
@@ -0,0 +1,12 @@
+server "api" {
+  api {
+    endpoint "/**" {
+      path = "/new/path/**"
+      proxy {
+        backend {
+          origin = "${env.COUPER_TEST_BACKEND_ADDR}/?q#f"
```
Comment (johakoch) [inline server/testdata/integration/api/08_couper.hcl:7]: An origin does not contain path, query or fragment.

**Comment by johakoch** [comment]
@alex-schneider ok for me now, please review my changes.

**Pull request merged**